entries/o/o2switch.fr.json

```
@@ -0,0 +1,14 @@
+{
+    "o2switch": {
+        "domain":"o2switch.fr",
+        "url": "https://clients.o2switch.fr",
+        "tfa": [
+            "email"
+        ],
+        "keywords": [
+            "hosting",
+            "domains"

```

```suggestion
            "hosting"
```

Other hosting providers that offer domains do not have the `domains` category